vectorizers/optimal_transport.py

```diff
@@ -850,16 +845,18 @@ def total_cost(flow, cost):
     return c
 
 
-@numba.njit(parallel=True)
+@numba.njit(nogil=True)
```

Comment:
Did you want to turn off parallelism here?

Reply:
Yeah, it wasn't actually gaining anything, and the inlining was happening anyway, so forcing it wasn't required.